vercel.json

```diff
@@ -1,5 +1,5 @@
 {
   "framework": null,
-  "buildCommand": "pnpm build",
+  "buildCommand": "npx convex deploy --cmd 'pnpm build'",
```

Comment:
a conditional for "if it's a preview deploy, don't deploy the convex functions" (if this project uses Vercel preview deploys)

Reply:
Is there a way you all typically go about this?

Reply:
I've done `if [ "$VERCEL_ENV" = "preview" ]; then npm run build; else npx convex deploy --cmd 'npm run build'; fi` before

Reply:
I was thinking it needs to be cross platform but if it's just in vercel.json I guess not